Eastern-Research-Group/expert-query
# Feature/259 update concatenated labels

## Related Issues:
* [EQ-259](https://jira.epa.gov/browse/EQ-259)

## Main Changes:
* Updated option labels that consist of ID and Name fields to take the format `ID **(Name)**`.
* Updated the field labels to match the same format.
  * I didn't update the label of the _State_ field, because it still refers to one underlying column.
* Added a semibold font-weight. This font variant _is_ included in USWDS, but the font-face isn't specified by default. Rather than referring to the font file in the USWDS `node_modules` path, I thought it would be safer to copy the file to the public folder.
* Moved `styles.css` to the public folder. This is done simply to simplify the path described above and to keep assets in one place. Also, if we ever choose to compile USWDS as [described in its documentation](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/), the output would be placed there. 

## Steps To Test:
1. Run the ETL locally.
2. Start the app, and go to http://localhost:3000/attains/assessments.
3. Confirm that options in the _State_ field display correctly when no options are selected, and after selecting a region.
4. Confirm options of other ID/Name fields display correctly.
